backend/tests/database/test_client.py

```
@@ -0,0 +1,209 @@
+import pytest
+from database._client import get_firestore, get_async_firestore, transaction
+from unittest.mock import patch, MagicMock, call
```

Ruff

`unittest.mock.call` imported but unused

Remove unused import: `unittest.mock.call`

(F401)